safegraph/delphi_safegraph/run.py

```diff
@@ -43,6 +46,7 @@ def run_module():
             'AWS_DEFAULT_REGION': aws_default_region,
         },
         shell=True,
+        check=True,
```

Comment:
What does this do?

Reply:
It causes an exception to be raised if the subprocess exits with non-zero status (as opposed to failing silently).  Pylint requested this be added.